docs/dotnet-ubuntu.md

```diff
@@ -0,0 +1,34 @@
+# Ubuntu .NET Core Versions
+
+.NET has changed the recommended install methods for Ubuntu from 2404.
+
+This document gives an overview of these change and the impact this has on the `runner-images`.
+
+## .NET Core for Ubuntu 2004 and 2024
```

Comment:
Shouldn't this be `2404` and not `2024` ?

Reply:
Yup, thanks :+1: Typo'd it should be 2204.

```suggestion
## .NET Core for Ubuntu 2004 and 2204
```